wdl-lint/src/rules/missing_requirements.rs

```diff
@@ -107,7 +107,15 @@ impl Visitor for MissingRequirementsRule {
                 if let Some(runtime) = task.runtime() {
                     let name = task.name();
                     state.exceptable_add(
-                        deprecated_runtime_section(name.as_str(), runtime.span()),
+                        deprecated_runtime_section(
+                            name.as_str(),
+                            runtime
+                                .syntax()
+                                .first_token()
```

Comment:
Does this narrow it down to just the `runtime` keyword? If so, you should update the `expect()` statement to make it clear that there is always at least one token, which is the `runtime` keyword.